tests/Integration/ORM/GenericEntityFactoryTest.php

```diff
@@ -55,6 +58,28 @@ public function can_use_factory_with_empty_constructor_without_persistence(): vo
         EmptyConstructorFactory::assert()->count(0);
     }
 
+    /**
+     * @test
+     */
+    #[Test]
+    public function generate_entity_with_uninitialized_field_with_proxy(): void
+    {
+        $entity = proxy_factory(EntityWithUninitializedFields::class)->create();
+
+        self::assertNull($entity->getUninitializedField());
```

Comment:
everything OK, because with the refresh, Doctrine correctly hydrates the property